src/navbar/menu.styles.tsx

```diff
@@ -0,0 +1,204 @@
+import styled from "styled-components";
+import { Color } from "../color";
+import { MediaQuery } from "../media";
+import { Text, TextStyleHelper } from "../text";
+
+// =============================================================================
+// STYLE INTERFACE, transient props are denoted with $
+// See more https://styled-components.com/docs/api#transient-props
+// =============================================================================
+interface StyleProps {
+    $selected: boolean;
+}
+
+// =============================================================================
+// WRAPPER
+// =============================================================================
+
+export const Wrapper = styled.ul`
+    display: flex;
+    list-style: none;
+    position: relative;
+
+    ${MediaQuery.MaxWidth.tablet} {
+        display: none;
+    }
+`;
+
+export const MobileWrapper = styled.ul`
+    display: none;
+    list-style: none;
+
+    ${MediaQuery.MaxWidth.tablet} {
+        border-left: 4px solid ${Color.Primary};
+        display: flex;
+        flex-direction: column;
+        padding-bottom: 1rem;
+    }
+`;
+
+// =============================================================================
+// LINK ITEMS
+// =============================================================================
+export const LinkItem = styled.li`
+    position: relative;
+    display: flex;
+    align-items: center;
+    :not(:last-of-type) {
+        margin-right: 1rem;
+    }
+
+    ${MediaQuery.MaxWidth.tablet} {
+        width: 100%;
+
+        :not(:last-of-type) {
+            margin-right: 0;
+        }
+    }
+`;
+
+export const Link = styled(Text.Hyperlink.Small)<StyleProps>`
+    display: flex;
+    position: relative;
+    align-items: left;
+    text-align: left;
+    color: ${Color.Neutral[1]};
+    font-style: normal;
+    font-weight: 400;
+    font-size: 14px !important;
+    line-height: 20px;
```

Comment:
use `${TextStyleHelper.getTextStyle("H6", "regular")}` instead to override the font styling. avoid using `!important`